scripts/debian/postinst.in

```diff
@@ -76,16 +76,18 @@ set -e
 echo "[POST-INSTALL] Trigger deamon-reload:"
 systemctl --system daemon-reload || true
 
-# If needed, try to load/compile the driver through falco-driver-loader
+# If needed, try to load/compile the driver through falcoctl
+echo "[POST-INSTALL] Configure falcoctl driver type:"
+falcoctl driver config --type $chosen_driver
```

Comment:
We now configure the desired driver type, and then build/download the driver using falcoctl.